base_layer/wallet/src/assets/asset_manager.rs

```diff
@@ -242,6 +242,34 @@ impl<T: OutputManagerBackend + 'static> AssetManager<T> {
             .await?;
         Ok((tx_id, transaction))
     }
+
+    pub async fn create_committee_checkpoint(
+        &mut self,
+        asset_public_key: PublicKey,
+        committee_pub_keys: Vec<PublicKey>,
+        effective_sidechain_height: u64,
+    ) -> Result<(TxId, Transaction), WalletError> {
+        let output = self
+            .output_manager
+            .create_output_with_features(
+                0.into(),
+                OutputFeatures::for_committee(
+                    asset_public_key,
+                    vec![2u8; 32],
+                    committee_pub_keys.clone(),
+                    effective_sidechain_height,
+                    true,
+                ),
+            )
+            .await?;
+
+        let (tx_id, transaction) = self
+            .output_manager
+            .create_send_to_self_with_output(vec![output], 100.into(), None, None)
```

Comment:
Fee_per_gram of 100 is a lot - the fees scale with the amount of metadata, outputs, inputs etc by themselves, so typically increasing the fee per gram isn't necessary. I see `create_registration_transaction` also has a fpg is 20 - I'd say that 5 -10 should probably be used everywhere unless a user sets it higher.

Reply:
Thanks @sdbondi well spotted, definitely agree